src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/NonAssociation.kt

```diff
@@ -151,6 +151,21 @@ data class DeleteNonAssociationRequest(
   val staffUserNameRequestingDeletion: String,
 )
 
+/**
+ * Request format to re-open a non-association between two prisoners
+ */
+@Schema(description = "Request to re-open a non-association")
+data class ReopenNonAssociationRequest(
+  @Schema(description = "Reason for re-opening the non-association", required = true, example = "Prisoners are fighting again", minLength = 1)
+  @field:Size(min = 1, message = "The reason for re-opening cannot be blank")
+  val reopenReason: String,
+  @Schema(description = "Date and time of the re-open, if not provided will default to today's time", required = false, example = "2023-06-07", defaultValue = "now")
```

Comment:
Shouldn't the example value be in the usual format and contains the time as well to avoid any possible confusion? e.g. something like `2021-12-31T12:34:56.789012`

Reply:
Updated